### PR TITLE
5/10

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

**Review by coffeegoddd — APPROVED** [review]
lgtm

**Review by max-hoffman — APPROVED** [review]
lgtm, only thing i can think of is maybe linking PR instead of saying json fix doesn't have a blog https://github.com/dolthub/dolt/pull/7785

**Comment by timsehn** [comment]
> lgtm, only thing i can think of is maybe linking PR instead of saying json fix doesn't have a blog [dolthub/dolt#7785](https://github.com/dolthub/dolt/pull/7785)

Good call!

File: emails/2024-05-10.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Now that Dolt's SQL is [fast](https://docs.dolthub.com/sql-reference/benchmarks/latency) and [correct](https://docs.dolthub.com/sql-reference/benchmarks/correctness), we have a lot more time available to work on making the Git-style version control features better. 
+
+This week, we're pleased to announce [a new puller](https://www.dolthub.com/blog/2024-05-08-dolt-new-puller/), the code that syncs databases between a remote and your local copy. It's a lot faster especially for home networks. 
+
+[![Faster Pulls](../images/faster-pulls.png)](https://www.dolthub.com/blog/2024-05-08-dolt-new-puller/)
+
+### Faster Index Builds. Also yes.
+
+We recently had a customer report a very slow index build. Upon digging in, we realized Dolt made some pretty strong assumptions about the hard drive it was attached to being SSD. This led us to fix a pretty [gnarly bug on our tablscan path](https://www.dolthub.com/blog/2024-05-06-hdd-read-planning/). This made index builds ~100X faster for large tables on hard disks.
```
Comment (fulghum) [inline emails/2024-05-10.md:15]:
```suggestion
We recently had a customer report a very slow index build. Upon digging in, we realized Dolt made some pretty strong assumptions about the hard drive it was attached to being SSD. This led us to fix a pretty [gnarly bug on our tablescan path](https://www.dolthub.com/blog/2024-05-06-hdd-read-planning/). This made index builds ~100X faster for large tables on hard disks.
```

**Pull request merged**